File: app/src/main/java/com/stratone/bmotion/model/Fuel.java
package com.stratone.bmotion.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Fuel {
    @SerializedName("FuelId")
    @Expose
    private Integer fuelId;
    @SerializedName("Name")
    @Expose
    private String name;
    @SerializedName("Price")
    @Expose
    private String price;
    @SerializedName("IsSubsidy")
    @Expose
    private String isSubsidy;
    @SerializedName("BackgroundColor")
    @Expose
    private String backgroundColor;
    @SerializedName("TextColor")
    @Expose
    private String textColor;
    @SerializedName("Liter")
    @Expose
    private int liter;

    public Integer getFuelId() {
        return fuelId;
    }

    public void setFuelId(Integer fuelId) {
        this.fuelId = fuelId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

    public String getIsSubsidy() {
        return isSubsidy;
    }

    public void setIsSubsidy(String isSubsidy) {
        this.isSubsidy = isSubsidy;
    }

    public String getBackgroundColor() {
        return backgroundColor;
    }

    public void setBackgroundColor(String backgroundColor) {
        this.backgroundColor = backgroundColor;
    }

    public String getTextColor() {
        return textColor;
    }

    public void setTextColor(String textColor) {
        this.textColor = textColor;
    }

    public int getLiter() {
        return liter;
    }

    public void setLiter(int liter) {
        this.liter = liter;
    }
}

File: BMotionServices/Models/Fuel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BMotionServices.Models
{
    public class Fuel
    {
        int fuelId;
        public int FuelId
        {
            get { return fuelId; }
            set { fuelId = value; }
        }

        string name;
        public String Name
        {
            get { return name; }
            set { name = value; }
        }

        string price;
        public String Price
        {
            get { return price; }
            set { price = value; }
        }

        string isSubsidy;
        public String IsSubsidy
        {
            get { return isSubsidy; }
            set { isSubsidy = value; }
        }

        string backgroundColor;
        public String BackgroundColor
        {
            get { return backgroundColor; }
            set { backgroundColor = value; }
        }

        string textColor;
        public String TextColor
        {
            get { return textColor; }
            set { textColor = value; }
        }
    }
}